examples/scaling_example/scaling.c

```diff
@@ -289,6 +291,16 @@ run_advanced_rings(struct onvm_nf_info *nf_info) {
         }
 
         while (keep_running && rx_ring && tx_ring && nf) {
+                /* Check for a stop message from the manager. */
```

Comment:
Put this after the packet processing, best to copy this order https://github.com/sdnfv/openNetVM-dev/blob/develop/onvm/onvm_nflib/onvm_nflib.c#L533

Reply:
We have a line before the actual packet processing that makes the loop continue if the number of packets is 0. Therefore, if I put the message check after that it won't work. Should we keep this check?